bot.ts

```diff
@@ -80,38 +95,51 @@ const commands: SlashCommandPartial[] = [
 		],
 	},
 	{
-		name: "source-code",
+		name: CommandNames.sourceCode,
 		description: "Get a link to the bot's source code",
-	}
+	},
 ];
 
 class SpeedrunCom extends ApplicationCommandsModule {
-	@slash("game-community")
+	// @slash(CommandNames.gameCommunity)
 	async gameCommunity(i: ApplicationCommandInteraction) {
 		const userMention = i.option("mention");
-		if (!userMention) {
-			await i.defer();
-		}
+		const gameAbbreviation = i.option("game");
+		const userId = userMention && userMention instanceof User
+			? userMention.id
+			: undefined;
+		const timeoutId = setTimeout(async () => await i.defer(), TIMEOUT_LIMIT_MS);
```

Comment:
```suggestion
		const timeoutId = setTimeout(i.defer, TIMEOUT_LIMIT_MS);
```

This makes it be consistant when defer is called on the other command

Reply:
For some reason I found `async () => await i.defer()` to work better than `i.defer`, and I don't really have an explanation for it. I can try again and make this and the other setTimeout consistent

Reply:
Yeah, I'm keeping this and added a comment to explain I don't know why